zerver/lib/narrow.py

```diff
@@ -179,6 +179,7 @@ def is_web_public_narrow(narrow: Optional[Iterable[Dict[str, Any]]]) -> bool:
 
 
 LARGER_THAN_MAX_MESSAGE_ID = 10000000000000000
+POSTGRESQL_INT_MAX = 2147483647
```

Comment:
Let's move this to be a property on `Message` as `Message.MAX_POSSIBLE_MESSAGE_ID` and place it next to the `id` field definition, because we will be eventually resizing the field to a bigint, and don't want to forget to adjust this when we do so.

Reply:
Updated!